### PR TITLE
Add comp lrc link

### PR DESCRIPTION
**Pull request by devenoges**
- update to addresses.json 1.1.2
 - more pax fixes
 - correct ustd current price tooltip
 - add COMP LRC & LINK

### CODE REVIEW

**Comment by nanexcool** [comment]
Thank you!

File: src/App.jsx [hunk 1]
```diff
@@ -229,6 +238,26 @@ class App extends Component {
...
+      [add.COMP, comp.interface.encodeFunctionData('totalSupply', [])],
+      [add.COMP, comp.interface.encodeFunctionData('balanceOf', [add.MCD_JOIN_COMP_A])], // 92
+      [add.MCD_FLIP_COMP_A, compAFlip.interface.encodeFunctionData('kicks', [])], // 93
+
+      [add.MCD_VAT, vat.interface.encodeFunctionData('ilks', [lrcAIlkBytes])], // 94
+      [add.MCD_JUG, jug.interface.encodeFunctionData('ilks', [lrcAIlkBytes])],
+      [add.MCD_SPOT, spot.interface.encodeFunctionData('ilks', [lrcAIlkBytes])], // 96
+      [add.LRC, lrc.interface.encodeFunctionData('totalSupply', [])],
+      [add.LRC, lrc.interface.encodeFunctionData('balanceOf', [add.MCD_JOIN_LRC_A])], // 98
+      [add.MCD_FLIP_COMP_A, lrcAFlip.interface.encodeFunctionData('kicks', [])], // 99
```
Comment (martinlsanchez) [inline src/App.jsx:253]: For sure this doesn't look right. 
I think it should be `add.MCD_FLIP_LRC_A,` @nanexcool

**Comment by martinlsanchez** [comment]
There is also something strange about auctions counters in the new collaterals. It's currently showing 2 Kicks each....

image

When actually, there was only ONE kick in each flipper contract (here a COMP FLIP link):

image
https://etherscan.io/address/0x524826F84cB3A19B6593370a5889A58c00554739#readContract

But for this, I could not found the fix... (maybe @nanexcool or @devenoges have some idea).

Hope this helps...

**Comment by devenoges** [comment]
thanks for the report, resolved by pr #78

**Pull request merged**